src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL57Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimeZone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1111</property>
```

Comment:
Just reminder not to forget in real projects to remove your credentials before pushing